spec/models/split_creation_spec.rb

```diff
@@ -41,6 +41,21 @@
     expect(weather.finished_at).to be_nil
   end
 
+  it 'undecides a decided split' do
```

Comment:
This will be important - to undo a decision you can simply reweight a split now. This will work over a migration or an admin panel reweight.

Reply:
As long as this won't allow us to change the semantics of an existing split thinking we're creating a new one. Like, if we decide a split and it's really only known to old mobile clients, and then in the future we reopen the split and accidentally enable something we shouldn't on those same old mobile clients...

Reply:
Yeah, I'm not crazy worried about that given our conventions around quarter-specific split naming. Also I'm going to do more in subsequent PRs (around supporting a new migration runner API) to shore up app ownership of splits so nobody can mutate another app's splits, which should keep spooky action at a distance down.